pkg/util/util.go

```diff
@@ -17,6 +17,18 @@ const HostRootMount = "/proc/1/root/"
 const CPUManagerOS3Path = HostRootMount + "var/lib/origin/openshift.local.volumes/cpu_manager_state"
 const CPUManagerPath = HostRootMount + "var/lib/kubelet/cpu_manager_state"
 
+const NonRootUID = 107
```

Comment:
Would it be possible to pass this UID as a cmdline arg to one of the virt-* binaries? Or maybe even discover it in runtime?

Reply:
Both options are possible but I would prefer to do it in follow-up PR. I thought about it before and I think it should be possible as a follow-up.

Reply:
Asking because having a const may not be very flexible. Also it may bring troubles if the environment changes. Would be great to have that in a follow-up PR. Thanks.